src/main/java/core/mate/academy/model/Machine.java

```diff
@@ -1,6 +1,9 @@
 package core.mate.academy.model;
 
-public abstract class Machine implements Workable {
+import core.mate.academy.service.MachineProducer;
+import java.util.List;
+
+public abstract class Machine implements Workable, MachineProducer {
```

Comment:
Why we need Machine class to implements MachineProducer? MachineProducer should produse some list of machines, but i dont think we need all Machines to have this behavior